shared/config/happy_config.go

```diff
@@ -391,10 +421,10 @@ func (s *HappyConfig) GetModuleSource() string {
 
 	if len(moduleSource) == 0 {
 		moduleSource = "git@github.com:chanzuckerberg/happy//terraform/modules/happy-stack-%s?ref=main"
-		if s.TaskLaunchType() == util.LaunchTypeK8S {
-			moduleSource = fmt.Sprintf(moduleSource, "eks")
-		} else {
+		if s.TaskLaunchType() == util.LaunchTypeFargate || s.TaskLaunchType() == util.LaunchTypeEC2 {
 			moduleSource = fmt.Sprintf(moduleSource, "ecs")
+		} else {
+			moduleSource = fmt.Sprintf(moduleSource, "eks")
 		}
```

Comment:
```suggestion
                 moduleSource = fmt.Sprintf(moduleSource, "eks")
		if s.TaskLaunchType() == util.LaunchTypeFargate || s.TaskLaunchType() == util.LaunchTypeEC2 {
			moduleSource = fmt.Sprintf(moduleSource, "ecs")
		}
```

Reply:
reduce unnecessary indents

Reply:
That won't work, as the substitution argument won't be available in the moduleSource.